composer.json

```diff
@@ -110,7 +110,7 @@
             "@stan"
         ],
         "test": "codecept build && codecept run unit",
-        "test-qa": "codecept build && codecept run acceptance",
+        "test-qa": "vendor/bin/phpcs --config-set default_standard Spryker && codecept build && codecept run acceptance",
```

Comment:
this should be using our default scheme:

cs-check and cs-fix

and then make sure to execute both in the yml file

Reply:
We need it to make acceptance tests work. For the CI, we use our default scheme https://github.com/spryker-sdk/sdk/blob/cf4de2c951e728e0f67e85f7a9a975e74ded3381/.github/workflows/ci.yml#L72, but without fix. Sorry, why should we have cs-fix in the CI yml file?

Reply:
No, that should only be in composer file and for local usage
But those are expected to be there for any non module package.

Reply:
https://github.com/spryker/decimal-object/blob/master/composer.json#L45-L46